synapse/cortex.py

```diff
@@ -1935,6 +1942,7 @@ def _initStormLibs(self):
         self.addStormLib(('user',), s_stormtypes.LibUser)
         self.addStormLib(('vars',), s_stormtypes.LibVars)
         self.addStormLib(('view',), s_stormtypes.LibView)
+        self.addStormLib(('macro',), s_stormlib_macro.LibMacro)
```

Comment:
pyramid

Reply:
depends on which element we're pyramiding on :)